.github/workflows/docs/conf.py

```diff
@@ -92,7 +91,8 @@
 def add_reference(
     app: Sphinx, env: BuildEnvironment, node: Element, contnode: TextElement
 ) -> Optional[nodes.reference]:
-    # Fix references in docstrings that are inherited from the base pytket.backends.Backend class.
+    # Fix references in docstrings that are inherited
```

Comment:
See above

Reply:
This change is manual to conform to the line width limit.